tests/test_image_caption.py

```diff
@@ -119,6 +119,7 @@ def test_article_content_image_keys(article_content, key):
                 "download_url": "https://example.com/image.jpg",
                 "name": "example_image.jpg",
                 "caption": "An example image",
+                "file_name": "data/raw/images/train/1.jpg",
```

Comment:
Det her burde løse problemet med Windows.

```suggestion
                "file_name": str(Path("data") / "raw" / "images" / "train" / "1.jpg"),
```